src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -177,19 +174,26 @@ typedef bmpl::vector<
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
     current<UsedParticleCurrentSolver>,
-    massRatio<MassRatioIons>,
-    chargeRatio<ChargeRatioIons>
-> ParticleFlagsIons;
+    massRatio<MassRatioElectrons>,
+    chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+    ,synchrotronPhotons<PIC_Photons>
+#endif
+#if(ENABLE_BREMSSTRAHLUNG == 1)
```

Comment:
I would prefer if @Heikman just creates a very small, new example, lets say a laser on a 20n_c foil, that uses a minimal set of input `.param` files to test Bremsstrahlung and synchrotron photons. That would avoid overloading our LWFA example even more and removes unnecessary `#define` macro magic.

You can also add a little notebook that then visualizes the expected output if you want with [openPMD-viewer](https://github.com/openPMD/openPMD-viewer).

Is that ok with you? :)